docs/specification/STU3BallotDraft.md

```diff
@@ -485,12 +496,12 @@ Field | Optionality | Type | Description
 
 All standard events are defined outside of the base FHIRcast specification in the Event Catalog with the single exception of the infrastructural `syncerror` event. 
 
-If the subscriber cannot follow the context of the event, for instance due to an error or a deliberate choice to not follow a context, the subscriber SHOULD communicate the error to the Hub in one of two ways.
+If the subscriber cannot follow the context of the event, for instance due to an error or a deliberate choice to not follow a context, the subscriber SHALL communicate the error to the Hub in one of two ways.
 
 * Responding to the event notification with an HTTP error status code as described in [Event Notification Response](#event-notification-response).
-* Responding to the event notification with an HTTP 202 (Accepted) as described above, then, once experiencing the error, send a `syncerror` event to the Hub. 
+* Responding to the event notification with an HTTP 202 (Accepted) as described above, then, once experiencing the error or refusing the change, send a `syncerror` event to the Hub. If the application cannot determine whether it will follow context within 10  10 seconds after reception of the event it SHOULD send a `syncerror` event.
```

Comment:
(remove the duplicate 10)
Is 10 seconds the appropriate time? The value is chosen to be same as heartbeat.

Add recommendation in syncreccomendation.md that when showing a modal, a syncerror should be send.